ext/standard/tests/general_functions/get_debug_type_basic.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Test get_debug_type() class reading
+--FILE--
+<?php
+
+namespace Demo {
+    class ClassInNamespace {
+
+    }
+}
+
+namespace {
+    class ClassInGlobal {
+
+    }
+
+    $anon = new class {
+
+    };
```

Comment:
Also add tests for `new class extends Whatever` or `new class implements Something`? As that's now included in the name.